vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -230,14 +225,16 @@ def _get_scheme_from_parts(
                     group_size=weight_quant.group_size)
 
         # Detect If Activation Quantization.
+        # TODO @dsikka: clean-up conditions
```

Comment:
Is this still a TODO?

Reply:
Yes. General follow-up on the state of these conditions